### ISSUE TITLE
Provide Link To Bats-Core

### ISSUE THREAD

**Issue by eggsbenjamin** [sstephenson/bats#275]
Hey @stefanlasiewski . This is a super useful tool and I'm glad to see that it's being actively developed at https://github.com/bats-core/bats-core. After encountering some problems running tests on first using bats it took me a little while to discover that I should be using bats-core instead. Would it be helpful to maybe provide a link at the top of the README? And/or archive this repo to make this explicit?

**Comment by stefanlasiewski** [comment]
Yes, both of those things would definitely be helpful. I think most of us found sstephenson/bats first, ran into problems, and then discovered bats-core/bats-core.

I think the following PR covers it: https://github.com/sstephenson/bats/pull/269. Also, see tickets which reference https://github.com/sstephenson/bats/issues/150.

The Maintainer isn't responding to requests in this repo, but I think we all should be compassionate about that, especially now in 2020. Life happens.

**Comment by bkhl** [comment]
It would indeed be helpful to archive this project and link to the new one, as this currently blocks updating Bats in some distributions, e.g: https://github.com/Homebrew/homebrew-core/issues/27796

**Comment by stefanlasiewski** [comment]
Luckily, it appears that Homebrew provides `bats-core` at https://formulae.brew.sh/formula/bats-core , even though the old `bats` is still present in Homebrew as well.
